config/src/main/resources/kiln.json

```diff
@@ -12,7 +12,7 @@
     "istanbulBlock": 0,
     "berlinBlock": 0,
     "londonBlock": 0,
-    "parisBlock": 1000,
+    "mergeNetSplitBlock": 1000,
```

Comment:
I'm assuming this is a guess, since merge was later than block 1000. I think this will break Kiln clients, not sure if we care...

Reply:
nethermind concurs:
https://github.com/NethermindEth/nethermind/pull/3805/files#diff-e279a74e5aeae3ea7401f86c8269cf9f2b40445b139302a65726662b10658832R25